### PR TITLE
info about process at proposal sent successfully page

### PR DESCRIPTION
**Pull request by benabraham**
(including added info about not sending an e-mail confirmation)
(https://github.com/pyvec/cz.pycon.org-2019/issues/27#issuecomment-456089942)

### CODE REVIEW

File: pyconcz/templates/default/proposals/proposal_success.html [hunk 1]
```diff
@@ -22,9 +22,13 @@ <h1>Application sent</h1>
             <h1>Proposal sent</h1>
 
             <p class="lead text-center">Thank you! <br>We’ve received your proposal.</p>
+
+            <p class="text-center font-weight-bold">Please note that you won’t receive any copy or confirmation of your proposal to your mailbox.</p>
+
             <p class="text-center">
                 All submissions will be scored after Call for Papers ends.
-                <br>The whole process takes some time and after it ends we let you know to the e-mail you’ve provided.
+                <br>The whole selection process takes usualy days.
```
Comment (hroncok) [inline pyconcz/templates/default/proposals/proposal_success.html]: and starts after the deadline?

Comment (frenzymadness) [reply pyconcz/templates/default/proposals/proposal_success.html]: Good point. It should be clear that the whole process may start after the deadline so nobody will expect some reply sooner.

**Comment by frenzymadness** [comment]
Are you gonna address a review or somebody else should do it?

**Comment by benabraham** [comment]
@hroncok happier?

**Comment by benabraham** [comment]
image

**Comment by hroncok** [comment]
What about this in big:

## Thank you! There is no automated e-mail, but we've received your proposal.

Scoring...

**Comment by benabraham** [comment]
🤷‍♂️

**Comment by frenzymadness** [comment]
Sorry, I missed Miro's comment. I think that your and Miro's wording are both clear enough so pick the one you prefer.

**Comment by benabraham** [comment]
@hroncok Thanks for thinking about this. It would definitely communicate this particular information very strongly but it isn't the most important part of what we want to say at the moment.

Merging as it is.

**Comment by benabraham** [comment]
Deployed to both beta and production.

**Pull request merged**